src/resources/js/components/UserDeleteModal.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { Modal } from 'flowbite-svelte';
+
+  import type { User } from 'bmlt-root-server-client';
+  import UserDeleteForm from './UserDeleteForm.svelte';
+
+  export let showUserDeleteModal: boolean;
+  export let selectedUser: User | null;
```

Comment:
This type shouldn't be nullable. The dialog will never be used without a `selectedUser`.

`export let selectedUser: User`